collect_app/src/main/java/org/odk/collect/android/formentry/FormEntryUseCases.kt

```diff
@@ -62,6 +66,17 @@ object FormEntryUseCases {
         )
     }
 
+    fun getSavePoint(formController: FormController, cacheDir: File): File? {
```

Comment:
We should be able to use this in other places we grab save point files, but I'll leave out that refactor for the moment to avoid the risk of merge conflicts that it'd introduce.